cmake/FindSndfile.cmake

```diff
@@ -0,0 +1,58 @@
+# Finds the libsndfile library
+
+# This file can be configured with the following CMake variables:
+# - Sndfile_ROOT_DIR
+# - Sndfile_INCLUDE_DIR
+# - Sndfile_LIBRARY
+
+# Or, alternatively, the following environment variables:
+# - sndfile_INSTALL_DIR
+# - LIBRARY_PATH
```

Comment:
I imagine by the time you see this comment I would've gotten my answer about comments like these.